packages/core/src/sanitization/sanitization.ts

```diff
@@ -117,7 +118,10 @@ export function ɵɵsanitizeResourceUrl(unsafeResourceUrl: any): TrustedScriptUR
   if (allowSanitizationBypassAndThrow(unsafeResourceUrl, BypassType.ResourceUrl)) {
     return trustedScriptURLFromStringBypass(unwrapSafeValue(unsafeResourceUrl));
   }
-  throw new Error('unsafe value used in a resource URL context (see https://g.co/ng/security#xss)');
+  const errorMessage = typeof ngDevMode === 'undefined' || ngDevMode ?
```

Comment:
I'd propose keeping parenthesis around the condition here (and in other cases in this PR where parenthesis are missing) for better readability:
```suggestion
  const errorMessage = (typeof ngDevMode === 'undefined' || ngDevMode) ?
```